core/sr-api-macros/tests/ui/changed_at_unknown_version.rs

```diff
@@ -0,0 +1,20 @@
+use runtime_primitives::traits::GetNodeBlockType;
```

Comment:
I probably failed by naming it `changed_in`, but either you rename the attribute or the file.